node/tests/reconfigure.rs

```diff
@@ -66,17 +66,30 @@ impl ExecutionState for SimpleExecutionState {
                 guard.epoch = epoch;
             };
 
-            let keypair = keys(None)
+            let k = keys(None);
+            let worker_cache = worker_cache_from_keys(&k);
+            let keypair = k
                 .into_iter()
                 .enumerate()
                 .filter(|(i, _)| i == &self.index)
                 .map(|(_, x)| x)
                 .collect::<Vec<_>>()
                 .pop()
                 .unwrap();
+
+            let worker_keypairs = keys(None);
```

Comment:
Yeah this probably will cause some issues where the worker keys need to be unique from each other and from their primary's keys. maybe when we generate worker keys we can use a seed to start or you can look at using the infra i just added here: https://github.com/MystenLabs/narwhal/pull/899

Reply:
I tried to use the new infra for this test but the test kept hanging (I believe it wasn't an issue with the keys but with how I was updating the committee epoch 🤷🏽 ). So I will just use the old method with a new seed for the worker keys to unblock this PR for now.

Reply:
Test is fixed now 👍🏽